src/app/static/src/app/utils.ts

```diff
@@ -281,4 +276,9 @@ export function getFilenameFromImportUrl(url: string) {
 export function getFilenameFromUploadFormData(formdata: FormData) {
     const file = formdata.get("file");
     return (file as File).name;
+}
+
+export enum HelpFile {
```

Comment:
Nice. I think we're expecting DE mode help files at some point, so we can expand this to include those then.